python/lsst/obs/lsstSim/selectImages.py

```diff
@@ -0,0 +1,99 @@
+    """Select images using their Wcs and a maximum seeing"""
+    ConfigClass = MaxPsfWcsSelectImageConfig
+    def runDataRef(self, dataRef, coordList, makeDataRefList=True, selectDataList=[]):
+        """Select images in the selectDataList that overlap the patch
+
+        We use the "convexHull" function in the geom package to define
+        polygons on the celestial sphere, and test the polygon of the
+        patch for overlap with the polygon of the image.
+
+        We use "convexHull" instead of generating a SphericalConvexPolygon
+        directly because the standard for the inputs to SphericalConvexPolygon
+        are pretty high and we don't want to be responsible for reaching them.
+        If "convexHull" is found to be too slow, we can revise this.
+
+        @param dataRef: Data reference for coadd/tempExp (with tract, patch)
+        @param coordList: List of Coord specifying boundary of patch
+        @param makeDataRefList: Construct a list of data references?
+        @param selectDataList: List of SelectStruct, to consider for selection
+        """
+        from lsst.geom import convexHull
+        psf_sizes = []
+        dataRefList = []
+        exposureInfoList = []
+
+        patchVertices = [coord.getVector() for coord in coordList]
+        patchPoly = convexHull(patchVertices)
+
+        for data in selectDataList:
+            dataRef = data.dataRef
+            imageWcs = data.wcs
+            cal = dataRef.get('calexp', immediate=True)
+            psf_size = cal.getPsf().computeShape().getDeterminantRadius()
+            nx,ny = data.dims
+
+            imageBox = afwGeom.Box2D(afwGeom.Point2D(0,0), afwGeom.Extent2D(nx, ny))
+            try:
+                imageCorners = [imageWcs.pixelToSky(pix) for pix in imageBox.getCorners()]
+            except (pexExceptions.DomainError, pexExceptions.RuntimeError) as e:
+                # Protecting ourselves from awful Wcs solutions in input images
+                self.log.debug("WCS error in testing calexp %s (%s): deselecting", dataRef.dataId, e)
+                continue
+
+            imagePoly = convexHull([coord.getVector() for coord in imageCorners])
+            if imagePoly is None:
+                self.log.debug("Unable to create polygon from image %s: deselecting", dataRef.dataId)
+                continue
+            if patchPoly.intersects(imagePoly): # "intersects" also covers "contains" or "is contained by"
+                psf_sizes.append(psf_size)
+                dataRefList.append(dataRef)
+                exposureInfoList.append(BaseExposureInfo(dataRef.dataId, imageCorners))
+        
+        filteredDataRefList = []
+        filteredExposureInfoList = []
+        for size, dataRef, expInfo in zip(psf_sizes, dataRefList, exposureInfoList):
+            # size is in sigma.  Convert to FWHM
+            if size*2.355 < self.config.maxPsfFwhm:
+                self.log.info("%s selected with FWHM of %f pixels"%(dataRef.dataId, size*2.355))
```

Comment:
Really minor, but I might have said

```
for size_sigma, dataRef, expInfo in zip(psf_sizes, dataRefList, exposureInfoList):
    # size is in sigma.  Convert to FWHM
    size_fwhm = 2.355*size_sigma
    if size_fwhm < self.config.maxPsfFwhm:
                self.log.info("%s selected with FWHM of %f pixels"%(dataRef.dataId, size_fwhm))
```

The comment may be superfluous at that point, but certainly doesn't hurt.